tutorials/data-lake/pull-dataset/README.md

```diff
@@ -1 +1,16 @@
-# Pull dataset from Rucio data lake
+# Interact with Rucio dataset files
+
+The following script assumes that all the files within a DID are present in a RSE (Rucio Storage Element), and that this RSE is accessible locally.
+ - DIDs are composed of a scope plus a dataset name in the `SCOPE:DataSet` format.
+ - If the files are not present in the RSE, replicate the dataset on the desired RSE before running the script.
+
+Run the following bash script
+
+```bash
+> ./rucio_dataset_files.sh <SCOPE:DataSet> <output_file> <output_symlink_dir>
```

Comment:
The script returns all the files within a dataset. The the user would interact with as many as he/she wants.

The script will generate two things
 - `<output_file>` contains the path to all the files within the dataset. These files are inside the `<output_symlink_dir>` folder.
 - `<output_symlink_dir>` a directory with all the files of the dataset. These are not real files, they are just symlinks to avoid duplicating the files within a storage. Also, it avoids the user to search within the storage element (what can get complicated depending on the site and type of storage).

Reply:
It's clear now. Thanks! Could you add these bullets to the docs please?